### PR TITLE
Add unfinished events to podium list

### LINKED ISSUE #46: Show empty podium for events instead of not showing event at all
An issue we ran into at Weston-super-Mare Open 2019 was quite funny. For Skewb and Square-1 we had both events as a Combined Final. However WCA Live auto-deletes competitors from previous rounds upon opening a subsequent round. Because of this, we did not manually remove the no-shows for these events. When it came to the award ceremony, it was exclaimed that we had missed out Square-1 and Skewb and we realised the reason immediately. 

I think once the first event has been completed, it would be good on the podium page to just show something along the lines of "**Undecided: Final round still in progress!**" under the events that don't have finalised rounds to avoid missing events.

### PR DESCRIPTION
**Pull request by saranshgrover**
Fixes #46

### CODE REVIEW

**Comment by jonatanklosko** [comment]
Could you please include a screenshot for the reference?

File: client/src/components/Podiums/Podiums.js [hunk 1]
```diff
@@ -56,6 +63,7 @@ const Podiums = ({ match }) => {
   if (loading && !data) return <Loading />;
   if (error) return <ErrorSnackbar />;
   const { competition } = data;
+  console.log(competition);
```
Comment (jonatanklosko) [inline client/src/components/Podiums/Podiums.js]: Nit: this shouldn't be here =)

**Comment by saranshgrover** [comment]
image

File: client/src/components/Podiums/Podiums.js [hunk 2]
```diff
@@ -75,6 +83,29 @@ const Podiums = ({ match }) => {
               />
             </Grid>
           ))}
+          {competition.events.length !== competition.podiums.length && (
+            <Grid item key={`events_not_completed`}>
```
Comment (jonatanklosko) [inline client/src/components/Podiums/Podiums.js]: The `key` prop is necessary only when rendering a list (usually with `Array#map`) and that's not that we do here =)

File: client/src/components/Podiums/Podiums.js [hunk 3]
```diff
@@ -75,6 +83,29 @@ const Podiums = ({ match }) => {
...
+              <Typography>{`Podiums for the following events are yet to be determined:`}</Typography>
+              <Grid container direction="row" spacing={3}>
+                {competition.events
+                  .filter(
+                    event =>
+                      competition.podiums.filter(
+                        podium => podium.event.id === event.id
+                      ).length === 0
```
Comment (jonatanklosko) [inline client/src/components/Podiums/Podiums.js]: As we test for the presence only, `.filter(...).length === 0` could be replaced with `.some(...)` :cat:

**Review by jonatanklosko — COMMENTED** [review]
Thanks for contributing with the clean implementation, just few comments =)

**Review by jonatanklosko — APPROVED** [review]
LGTM, thanks! :tada:

**Pull request merged**